feat(trace-view): Add tests for the light trace endpoint

wmak opened the pull request:
- This adds tests that were originally skipped in #23701 so that we
  could develop this feature more quickly

wmak committed:
feat(trace-view): Add tests for the light trace endpoint

- This adds tests that were originally skipped in #23701 so that we
  could develop this feature more quickly

wmak commented on tests/snuba/api/endpoints/test_organization_events_trace.py:
```diff
@@ -0,0 +1,268 @@
+                    }
+                ],
+                parent_span_id=root_span_id,
+            )  # todo different project
```
gonna do this todo before, undrafting the PR

markstory commented on tests/snuba/api/endpoints/test_organization_events_trace.py:
```diff
@@ -0,0 +1,312 @@
+                    {
+                        "same_process_as_parent": True,
+                        "op": "http",
+                        "description": "GET /api/plants/?all_plants=1",
```
Should these change between generations?

wmak committed:
ref: Give spans different names

Zylphrex commented on tests/snuba/api/endpoints/test_organization_events_trace.py:
```diff
@@ -0,0 +1,312 @@
+            trace=trace,
+            spans=spans,
+        )
+        data["timestamp"] = iso_format(before_now(minutes=1))
+        data["start_timestamp"] = iso_format(before_now(minutes=5))
```
nit: You can add both `timestamp` and `start_timestamp` to `load_data` instead of manually setting it here.

Zylphrex commented on tests/snuba/api/endpoints/test_organization_events_trace.py:
```diff
@@ -0,0 +1,312 @@
+                format="json",
+            )
+
+        assert response.status_code == 400, response.content
```
Should this also be a 404?

Zylphrex commented on tests/snuba/api/endpoints/test_organization_events_trace.py:
```diff
@@ -0,0 +1,312 @@
+                {
+                    "same_process_as_parent": True,
+                    "op": "http",
+                    "description": "GET span_generation_1",
```
Do we want to have unique span descriptions here?

wmak committed:
ref: Addressing PR comments

-  404 when the event_id passed can't be found
- Giving transactions and spans better names.... i hate this
  `zip(enumerate` syntax...

Zylphrex reviewed (approved):
LGTM
